ai/integrations/gemini/README.md

```diff
@@ -0,0 +1,179 @@
+)
+
+# Initialize the asynchronous ApiClient
+api_client = ApiClient(configuration=config)
+
+# Instantiate the UnitycatalogFunctionClient
+uc_client = UnitycatalogFunctionClient(api_client=api_client)
+
+# Example catalog and schema names
+CATALOG = "my_catalog"
+SCHEMA = "my_schema"
+```
+
+### Creating a Unity Catalog Function
+
+You can create a UC function either by providing a Python callable or by submitting a `FunctionInfo` object. Below is an example (recommended) of using the `create_python_function` API that accepts a Python callable (function) as input.
+
+To create a UC function from a Python function, define your function with appropriate type hints and a Google-style docstring:
+
+```python
+def add_numbers(a: float, b: float) -> float:
+    """
+    Adds two numbers and returns the result.
+
+    Args:
+        a (float): First number.
+        b (float): Second number.
+
+    Returns:
+        float: The sum of the two numbers.
+    """
+    return a + b
+
+# Create the function within the Unity Catalog catalog and schema specified
+function_info = uc_client.create_python_function(
+    func=add_numbers,
+    catalog=CATALOG,
+    schema=SCHEMA,
+    replace=False,  # Set to True to overwrite if the function already exists
+)
+
+print(function_info)
+```
+
+## Databricks-managed Unity Catalog
+
+To use Databricks-managed Unity Catalog with this package, follow the [instructions](https://docs.databricks.com/en/dev-tools/cli/authentication.html#authentication-for-the-databricks-cli) to authenticate to your workspace and ensure that your access token has workspace-level privilege for managing UC functions.
+
+### Client setup
+
+Initialize a client for managing UC functions in a Databricks workspace, and set it as the global client.
+
+```python
+from unitycatalog.ai.core.base import set_uc_function_client
+from unitycatalog.ai.core.databricks import DatabricksFunctionClient
+
+client = DatabricksFunctionClient(
+    warehouse_id="..." # replace with the warehouse_id
+    cluster_id="..." # optional, only pass when you want to use cluster for function creation
+)
+
+# sets the default uc function client
+set_uc_function_client(client)
+```
+
+### Create a Function in UC
+
+Create a python UDF in Unity Catalog with the client
+
+```python
+# replace with your own catalog and schema
+CATALOG = "catalog"
+SCHEMA = "schema"
+
+func_name = f"{CATALOG}.{SCHEMA}.python_exec"
+# define the function body in SQL
+sql_body = f"""CREATE OR REPLACE FUNCTION {func_name}(location STRING COMMENT 'Retrieves the current weather from a provided location.')
+RETURNS STRING
+LANGUAGE PYTHON
+COMMENT 'Returns the current weather from a given location and returns the temperature in degrees Celsius.'
+AS $$
+    return "31.9 C"
+$$
+"""
+
+client.create_function(sql_function_body=sql_body)
+```
+
+Now that the function is created and stored in the corresponding catalog and schema, we can use it within Anthropic's SDK.
```

Comment:
```suggestion
Now that the function is created and stored in the corresponding catalog and schema, we can use it within Gemini's SDK.
```

There are other anthropic references below

Reply:
Will edit and make sure no reference are present